src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,99 @@ $pf-chart-color-black-200: $pf-color-black-400;
+$pf-chart-global-layout--Height: 300px;
+$pf-chart-global-layout--Width: 450px;
+
+// Individual Charts
+
+// Area Chart
+$pf-chart-area--Opacity: .4;
+$pf-chart-area-stroke--Width: 2px;
+
+// Axis Chart
+$pf-chart-axis-axis-stroke--Width: 1px;
+$pf-chart-axis-axis-label--Padding: 100px;
+$pf-chart-axis--ticks--Width: 1px;  // this should be a var for size so not sure if width is appropriate
+
+// Bar Chart
+$pf-chart-bar--Width: 10px;
+
+// Box Plot Chart
+$pf-chart-boxplot-max--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-max-stroke--Width: 1px;
+$pf-chart-boxplot-median--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-median-stroke--Width: 1px;
+$pf-chart-boxplot-min--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-min-stroke--Width: 1px;
+$pf-chart-boxplot-q1--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-q3--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-box--Width: 20px;
+
+// Bullet Chart
+$pf-chart-bullet--comparative-measure-line-warning--BorderColor: $pf-color-orange-300;
+$pf-chart-bullet--comparative-measure-line-warning--BorderWidth: 1px;
+$pf-chart-bullet--comparative-measure-line-error--BorderColor: $pf-color-red-100;
+$pf-chart-bullet--comparative-measure-line-error--BorderWidth: 1px;
+$pf-chart-bullet--zero-line--BorderColor: $pf-color-black-700;
+$pf-chart-bullet--zero-line--BorderWidth: 1px;
+$pf-chart-bullet--negative-primary-measure--BorderColor: $pf-color-red-100;
+$pf-chart-bullet--negative-primary-measure--BorderWidth: 2px;
+
+// Candlestick
+$pf-chart-candelstick-data-stroke--Width: 1px;
+
+// Chart
+$pf-chart-chart--BorderWidth: 1px;
+
+// Donut Chart
+$pf-chart-donut-color-threshold-first: $pf-color-black-200;
+$pf-chart-donut-color-threshold-second: $pf-color-black-300;
+$pf-chart-donut-color-threshold-third: $pf-color-black-400;
+$pf-chart-donut-color-threshold-success: $pf-chart-color-blue-300;
+$pf-chart-donut-color-threshold-warning: $pf-chart-color-gold-400;
+$pf-chart-donut-color-threshold-danger: $pf-color-red-100;
+
+// Error Bar
+$pf-chart-errorbar--BorderWidth: $pf-global--spacer--sm;
+$pf-chart-errorbar-data-stroke--Width: 2px;
+
+// Legend
+$pf-chart-legend-gutter--Width: 20px;
+$pf-chart-legend-title--FontSize: $pf-chart-global--FontSize;
+$pf-chart-legend-title--Padding: 2px;
+
+// Line Chart
+$pf-chart-line-stroke--Width: 2px;
+
+// Pie Chart
+$pf-chart-pie--Padding: $pf-global--spacer--sm;
+$pf-chart-pie-data-Padding: $pf-global--spacer--sm;
+$pf-chart-pie-data-stroke--Width: 1px;
+$pf-chart-pie-labels--Padding: $pf-global--spacer--sm;
+$pf-chart-pie--Height: 230px;
+$pf-chart-pie--Width: 230px;
+
+// Scatter Chart
+$pf-chart-stack-data-stroke--Width: 1px;
+
+// Tooltip
+$pf-chart-tooltip-pointer--Height: 10px; // this is equivalent to length
+$pf-chart-tooltip-pointer--Width: 20px;
+$pf-chart-tooltip--Padding: $pf-global--spacer--md;
+
+// Voronoi Chart
+$pf-chart-voronoi-label--Padding: $pf-global--spacer--sm;
+$pf-chart-voronoi-flyout-stroke--Width: 1px;
```

Comment:
Missing a few important props here. For example, padding, strokeWdith, etc., which are different than the flyout tooltip styles

![Screen Shot 2019-05-29 at 11 50 49 AM](https://user-images.githubusercontent.com/17481322/58571582-0a8ef100-8208-11e9-9380-ffbfc1ed7a28.png)